docs/proposals/scheduling/20220629-schedule-device-in-card-level.md

```diff
@@ -0,0 +1,143 @@
+---
+title: schedule device(gpu\rdma) in card level
+authors:
+- "@buptcozy"
+reviewers:
+- "@hormes"
+- "@allwmh"
+- "@honpey"
+- "@jasonliu747"
+- "@saintube"
+- "@stormgbs"
+- "@zwzhang0107"
+creation-date: 2022-06-29
+last-updated: 2022-06-29
+status: provisional
+
+---
+
+# schedule device in card-level
+
+## Table of Contents
+
+<!--ts-->
+
+* [schedule device in card level](#schedule-device-in-card-level)
+  * [Table of Contents](#table-of-contents)
+  * [Summary](#summary)
+  * [Motivation](#motivation)
+  * [Goals](#goals)
+  * [User Stories](#user-stories)
+    * [Story NVLink](#story-NVLink)
+    * [Story MIG](#story-MIG)
+    * [Story Oversold](#story-Oversold)
+  * [Implementation Details](#implementation-details)
+    * [scheduler](#scheduler)
+    * [device reporter](#device-reporter)
+    * [RuntimeProxy](#runtimeProxy)
+
+<!--te-->
+
+## Summary
+support schedule devices like gpu\rdma\fpga in card level for better schedule placement. 
+
+## Motivation
+there is lots of cases that need scheduler make better scheule placement with devices like gpu\rdma etc.
+like gpu-nvlink in A100\V100、gpu Mig in A100.
+
+### Goals
+1、a new program on can report card level device crd, which will not conflict with kubelet and existed device plugin
```

Comment:
a new program on node ?

Reply:
why do you need a new reporter when there is koordlet?